src/BinOp.chpl

```diff
@@ -109,6 +109,9 @@ module BinOp
           when ">=" {
             e.a = l.a:int >= r.a:int;
           }
+          when "+" {
+            e.a = (l.a:int + r.a:int):bool;
+          }
```

Comment:
similarly I don't think we need casts. It seems `e.a = l.a | r.a` should do what we want

Reply:
Yes, this is much better than the casting. Changed